GeneBe.pm

```diff
@@ -54,6 +57,19 @@ use Bio::EnsEMBL::Variation::Utils::Sequence qw(get_matched_variant_alleles);
 
 use base qw(Bio::EnsEMBL::Variation::Utils::BaseVepPlugin);
 
+sub new {
+  my $class = shift;
+  
+  my $self = $class->SUPER::new(@_);
+
+  $self->get_user_params();
+  my $param_hash = $self->params_to_hash();
+
+  $self->{user} = $param_hash->{user} if defined $param_hash->{user};
+  $self->{password} = $param_hash->{password} if defined $param_hash->{password};
+
+}
```

Comment:
Pleasure is mine. I was testing the changes. There is still one changes you need to add. Sorry it was on my part - I have you some incomplete code block.

```suggestion

  my $param_hash = $self->params_to_hash();

  $self->{user} = $param_hash->{user} if defined $param_hash->{user};
  $self->{password} = $param_hash->{password} if defined $param_hash->{password};

  return $self;
}
```